src/test/java/pagesTests/BaseClass.java

```diff
@@ -0,0 +1,39 @@
+package pagesTests;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import com.seleniumWebDriver.pages.LogInPage;
+
+import java.time.Duration;
+
+public abstract class BaseClass {
```

Comment:
Why tests are not in org.seleniumWebDriver package ?
the better name of folder will be tests instead of pagesTests. You are not testing pages , your are testing application which was splitted by you on pages

Reply:
Reworked